Doc/library/stdtypes.rst

```diff
@@ -1406,12 +1406,21 @@ The following methods are present only on unicode objects:
    that have the Unicode numeric value property, e.g. U+2155,
    VULGAR FRACTION ONE FIFTH.
 
+.. method:: unicode.isdigit()
```

Comment:
Wouldn't it make more sense to put this next to the _str.isdigit_ method, like is done with the _splitlines_ methods? It doesn't really belong in this section.